sdk/storage/storage-file/test/node/highlevel.node.spec.ts

```diff
@@ -56,8 +51,8 @@ describe("Highlevel", () => {
     fs.unlinkSync(tempFileSmall);
   });
 
-  it("uploadFileToAzureFile should success for large data", async () => {
-    await uploadFileToAzureFile(tempFileLarge, fileClient, {
+  it("uploadFile should success for large data", async () => {
```

Comment:
Where does the browser highlevel test cases go?

Reply:
I didn't see one. Don't see browser under https://github.com/Azure/azure-storage-js/tree/master/file/tests either.